guides/plugins/plugins/framework/message-queue/handle-asynchronous-messages.md

```diff
@@ -0,0 +1,57 @@
+---
+nav:
+  title: Handle asynchronous messages
+  position: 40
+
+---
+
+# Handle asynchronous messages
+
+## Overview
+
+In this guide you will learn how to handle asynchronous messages and the difference between the asynchronous queues.
+
+## Prerequisites
+
+This guide is built upon the [Add message to queue guide](add-message-to-queue), so you should have read that first.
+
+## Different queues
```

Comment:
can you link to the original symfony docs for multiple queues, as we use the symfony default and hosters could modify this to their needs

Reply:
I will add a link to the [prioritized transports](https://symfony.com/doc/current/messenger.html#prioritized-transports) documentation because this is also the reason, why we have multiple queues :smile:

Reply:
For me personally it would have been sufficient to add the information about implementing the `AsyncLowPriorityMessageInterface` on the existing article "Add message to queue" 😅